.vsts-dotnet-ci.yml

```diff
@@ -86,7 +86,7 @@ jobs:
     pool:
       name: NetCore-Public
       demands: ImageOverride -equals build.ubuntu.1804.amd64.open
-    helixQueue: Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-mlnet-helix
+    helixQueue: Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8-mlnet
```

Comment:
This might need to be `centos-stream8-helix` or we may need to create a mlnet-helix specific image if we have runtime dependencies that differ from those used by dotnet/runtime.  I see the runtime does some specific setup for helix, and our MLNet image installs OpenMP

Reply:
Yeah, tests got surprisingly far - most passing - just failed to run python reporting script.
https://[helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-machinelearning-refs-pull-6703-merge-4564591d5fa74c8696/Microsoft.ML.Core.Tests/1/console.bbe2343b.log?helixlogtype=result](https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-machinelearning-refs-pull-6703-merge-4564591d5fa74c8696/Microsoft.ML.Core.Tests/1/console.bbe2343b.log?helixlogtype=result)
https://[helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-machinelearning-refs-pull-6703-merge-4564591d5fa74c8696/Microsoft.ML.IntegrationTests/1/console.1e6d4315.log?helixlogtype=result](https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-machinelearning-refs-pull-6703-merge-4564591d5fa74c8696/Microsoft.ML.IntegrationTests/1/console.1e6d4315.log?helixlogtype=result)
https://[helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-machinelearning-refs-pull-6703-merge-4564591d5fa74c8696/Microsoft.Data.Analysis.Interactive.Tests/1/console.98a8585a.log?helixlogtype=result](https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-machinelearning-refs-pull-6703-merge-4564591d5fa74c8696/Microsoft.Data.Analysis.Interactive.Tests/1/console.98a8585a.log?helixlogtype=result)

I'll try with the runtime container.  If that doesn't work I'll try to make a new container with our pre-reqs + helix's.

Reply:
Yep - we need a new container image here.   `centos-stream8-mlnet` is missing python, and `centos-stream8-helix` is missing ML.NET's openmp dependency.

Reply:
https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/942